File: BIPClient/BIPFramework/core/ParameterInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace com.ccf.bip.framework.core
{
    /// <summary>
    /// 平台服务调用统一参数信息
    /// </summary>
    [Serializable]
    public class ParameterInfo
    {
        private String serviceName;
        /// <summary>
        /// 调用服务名称
        /// </summary>
        public String ServiceName
        {
            get { return serviceName; }
            set { serviceName = value; }
        }
        private String functionName;
        /// <summary>
        /// 调用服务方法名称
        /// </summary>
        public String FunctionName
        {
            get { return functionName; }
            set { functionName = value; }
        }
        private Object[] value;
        /// <summary>
        /// 调用服务方法参数
        /// </summary>
        public Object[] Value
        {
            get { return this.value; }
            set { this.value = value; }
        }
    }
}

File: BIPServer/src/com/ccf/bip/framework/core/ParameterInfo.java
package com.ccf.bip.framework.core;

import java.io.Serializable;

public class ParameterInfo implements Serializable {
    private static final long serialVersionUID = 1791430594578320471L;

    private String serviceName;
    private String functionName;
    private Object[] value;
    private String tocken;
    private boolean sessionUpdate = true;//请求是否更新session，默认为true
    
    public String getServiceName() {
        return serviceName;
    }
    public void setServiceName(String serviceName) {
        this.serviceName = serviceName;
    }
    public String getFunctionName() {
        return functionName;
    }
    public void setFunctionName(String functionName) {
        this.functionName = functionName;
    }
    public Object[] getValue() {
        return value;
    }
    public void setValue(Object[] value) {
        this.value = value;
    }
	public String getTocken() {
		return tocken;
	}
	public void setTocken(String tocken) {
		this.tocken = tocken;
	}
	public boolean isSessionUpdate() {
		return sessionUpdate;
	}
	public void setSessionUpdate(boolean sessionUpdate) {
		this.sessionUpdate = sessionUpdate;
	}    
}
